syntax = "proto3";

package rj_ChatServer;

message MsgRequest
{
    int32 from_id = 1;
    int32 to_id = 2;
    bytes msg = 3;
}

message GroupRequest
{
    int32 from_id = 1;
    int32 group_id = 2;
    bytes msg = 3;
}